libraries/plugins/chain/chain_plugin.cpp

```diff
@@ -54,6 +57,8 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
    cli.add_options()
          ("replay-blockchain", bpo::bool_switch()->default_value(false), "clear chain database and replay all blocks" )
          ("resync-blockchain", bpo::bool_switch()->default_value(false), "clear chain database and block log" )
+         ("stop-replay-at-block", bpo::value<uint32_t>(), "Stop and exit after reaching given block number")
+         ("collect-benchmark-results", bpo::value<uint32_t>(), "Print time and memory usage every given number of blocks")
```

Comment:
This option name did not make sense to me until I read the description. Perhaps `benchmark-interval` would be a better name?

Reply:
Just saw that's what you called the variable. Yeah, let's use this name.